src/CalculatorWADTests/ScenarioStandard.cs

```diff
@@ -0,0 +1,160 @@
+            session.FindElementByXPath("//Button[@AutomationId=\"num9Button\"]").Click();
+            session.FindElementByXPath("//Button[@AutomationId=\"minusButton\"]").Click();
+            session.FindElementByXPath("//Button[@AutomationId=\"num1Button\"]").Click();
+            session.FindElementByXPath("//Button[@AutomationId=\"equalButton\"]").Click();
+            Assert.AreEqual("8", GetCalculatorResultText());
+        }
+
+        /*
+        [TestMethod]
+        public void Templatized()
+        {
+            Templatized("One", "Plus", "Seven", "8");
+            Templatized("Nine", "Minus", "One", "8");
+            Templatized("Eight", "Divide by", "Eight", "1");
+        }
+        */
+        [TestMethod]
+        [DataRow("One", "Plus", "Seven", "8")]
+        [DataRow("Nine", "Minus", "One", "8")]
+        [DataRow("Eight", "Divide by", "Eight", "1")]
+        public void Templatized(string input1, string operation, string input2, string expectedResult)
+        {
+            Thread.Sleep(5000);
+            // Run sequence of button presses specified above and validate the results
+            session.FindElementByName(input1).Click();
+            session.FindElementByName(operation).Click();
+            session.FindElementByName(input2).Click();
+            session.FindElementByName("Equals").Click();
+            Assert.AreEqual(expectedResult, GetCalculatorResultText());
+        }
+        
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            // Create session to launch a Calculator window
+            Setup(context);
+
+            Thread.Sleep(5000);
+
+            // Identify calculator mode by locating the header
+            try
+            {
+                header = session.FindElementByAccessibilityId("Header");
+            }
+            catch
+            {
+                header = session.FindElementByAccessibilityId("ContentPresenter");
+            }            
+
+            // Ensure that calculator is in standard mode
+            if (!header.Text.Equals("Standard", StringComparison.OrdinalIgnoreCase))
+            {
+                session.FindElementByAccessibilityId("TogglePaneButton").Click();
+                Thread.Sleep(TimeSpan.FromSeconds(1));
+                var splitViewPane = session.FindElementByClassName("SplitViewPane");
+                splitViewPane.FindElementByName("Standard Calculator").Click();
+                Thread.Sleep(TimeSpan.FromSeconds(1));
+                Assert.IsTrue(header.Text.Equals("Standard", StringComparison.OrdinalIgnoreCase));
+            }
+
+            Thread.Sleep(5000);
+
+            // Locate the calculatorResult element
+            calculatorResult = session.FindElementByAccessibilityId("CalculatorResults");
+            Assert.IsNotNull(calculatorResult);
+        }
+
+        [ClassCleanup]
+        public static void ClassCleanup()
+        {
+            TearDown();
```

Comment:
nit: base.TearDown

Reply:
Same reason for why "base" cannot be used as above for "this".

Reply:
whoops